config/software/ruby.rb

```diff
@@ -247,6 +247,11 @@
     # https://github.com/wayneeseguin/rvm/commit/86766534fcc26f4582f23842a4d3789707ce6b96
     configure_command << "ac_cv_func_dl_iterate_phdr=no"
     configure_command << "--with-opt-dir=#{install_dir}/embedded"
+  elsif solaris2? && sparc? && solaris_11? && version.satisfies?(">= 2.5")
```

Comment:
i'd prefer just `if solaris2?` here.  i know it only failed on sparc on 11 on ruby-2.5.0 but i really never want to see this api on solaris again...

Reply:
Works for me.